server/src/test/java/io/crate/metadata/upgrade/IndexTemplateUpgraderTest.java

```diff
@@ -27,13 +27,16 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.contains;
 import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertFalse;
```

Comment:
still not addressed.

Reply:
hmm, probably lost while rebasing... And sorry I should have set this to back draft